SOURCES/component-status-cfn-signal.sh

```diff
@@ -19,4 +19,4 @@ done
 
 # We should only reach here if the check above has confirmed that the app is up
 # resource should be set to the Logical Resource id of the instances AutoScalingGroup
-/usr/local/bin/cfn-signal --stack="${STACKID}" --region="${REGION}" --resource="ComponentAutoScalingGroup" --success=true
\ No newline at end of file
+/usr/local/bin/cfn-signal --stack="${STACKID}" --region="${REGION}" --resource="ApplicationAutoScalingGroup" --success=true
```

Comment:
Should this be `AutoScalingGroup`? Is there a reference to `ApplicationAutoScalingGroup` somewhere?

Reply:
https://github.com/bbc/frameworks-infra/blob/master/mozart/mozart-fetcher/stacks/main.json#L404